resources/views/livewire/import-youtube-live-stream.blade.php

```diff
@@ -27,7 +27,15 @@
                                     <input type="text" wire:model.defer="youtubeId" id="youtubeStreamId"
                                            class="mt-1 focus:ring-indigo-500 focus:border-indigo-500 block w-full shadow-sm sm:text-sm border-gray-300 rounded-md">
                                 </div>
-
+                                <div class="col-span-6 sm:col-span-3">
```

Comment:
Please add the language `behind` the date for now. We can still work on design improvements later if needed.

Reply:
Fixed!